t/plugin/clickhouse-logger.t

```diff
@@ -149,7 +161,7 @@ property "endpoint_addr" is required
                                 "password": "a",
                                 "database": "default",
                                 "logtable": "t",
-                                "endpoint_addr": "http://127.0.0.1:10420/clickhouse-logger/test",
+                                "endpoint_addrs": ["http://127.0.0.1:10420/clickhouse-logger/test"],
```

Comment:
We can keep an old test to test the compatibility of the deprecated configuration

Reply:
ok